PrebidMobile/ConfigurationAndTargeting/Prebid.swift

```diff
@@ -110,7 +110,6 @@ public class Prebid: NSObject {
     public var creativeFactoryTimeoutPreRenderContent: TimeInterval = 30.0
     
     //Controls whether to use PrebidMobile's in-app browser or the Safari App for displaying ad clickthrough content.
-    //For original API should be true
     public var useExternalClickthroughBrowser = false
```

Comment:
This property looks outdated now. 

**Rendering API**. SDK opens an URL only in the `SFSafariViewController`, right? It means the `imp.clickbrowser` will always be `1`. 

**Original API**. This scenario assumes rendering in any ad server SDK. We can't know which browser will be used. But we should give publishers ability to clarify the behavior if needed. Since the Original API is mostly ised with GMA SDK, we can assume that the click URL will be opened in  `SFSafariViewController` as well. So the default value should be `1` too. 

The name is irrelevant anymore too. There is no attempt to open an URL in something _external_. Let's rename this property to something more OpenRTB-focused, like `impClickbrowser` with default value `@1` and ability to customize it only for Original API integration. The comment to his property should point to the description of the respective field in the OpenRTB standard.